xlson/__init__.py

```diff
@@ -225,12 +225,18 @@ def __init__(self, **kwargs: Dict) -> None:
 
         params["keys"] = [child["name"] for child in params[CHILDREN]]
         params["values"] = [child["label"] for child in params[CHILDREN]]
-        choice_ids_options = [
-            child["instance"]["openmrs_entity_id"] for child in params[CHILDREN]
-        ]
-        params["openmrs_choice_ids"] = {
-            k: v for k, v in zip(params["keys"], choice_ids_options)
-        }
+
+        try:
+
+            choice_ids_options = [
+                child["instance"]["openmrs_entity_id"] for child in params[CHILDREN]
+            ]
+            params["openmrs_choice_ids"] = {
+                k: v for k, v in zip(params["keys"], choice_ids_options)
+            }
+        except KeyError as error:
+            error.args += ("Missing  openmrs_entity_id in choices sheet",)
```

Comment:
Could we simply log the error message as opposed to building a custom error message on top of it?